packages/foreman/rubygem-apipie-dsl/rubygem-apipie-dsl.spec

```diff
@@ -1,64 +1,43 @@
-# template: scl
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name apipie-dsl
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 2.5.0
+Name: rubygem-%{gem_name}
+Version: 2.6.0
 Release: 1%{?dist}
 Summary: Ruby DSL documentation tool
-Group: Development/Languages
 License: MIT
 URL: https://github.com/ofedoren/apipie-dsl
```

Comment:
This is taken from the gem metadata, but that should be updated to https://github.com/Apipie/apipie-dsl.

Reply:
Ohhh... I forgot to update it in the gem after I've transferred it under Apipie org :/ I'll update here, and there, but I hope we don't need another release just for that?

Reply:
No, only if we regenerate the spec file. We don't do that often. I'll just happen naturally when there's a new release.